### ISSUE TITLE
Specify Display in newest Driver version

### ISSUE THREAD

**Issue by Seneral** [ValveSoftware/openvr#992]
https://steamcommunity.com/app/358720/discussions/0/1743356969850162401/

I'm making my own Driver with the newest OpenVR version 1.1.3 and I'm trying to let SteamVR know which monitor is my HMD monitor. Up until 1.0.6, IClientTrackedDeviceProvider.SetDisplayId was responsible for this, but it has been silently removed, the documentation has not been updated to reflect this and there are no obvious replacements that I have found.

The closest I've got is Prop_EdidVendorID_Int32 and Prop_EdidProductID_Int32 that seem to be responsible for identifying the display, but even though they were introduced in 0.9.12, there is no documentation or even a note in the 0.9.12 (or 0.9.10) update explaining their use.

Since [EDID ](https://en.wikipedia.org/wiki/Extended_Display_Identification_Data) has no defined values for VendorID and ProductID, only ManufacturerID and Manufacturer Product Code, it is not clear which values these properties need to have in order to identify a display.

I've tried variations of the ManufacturerID (big endian, low endian) as VendorID and the Manufacturer product code as ProductID, but that does not seem to make a difference to the errors I get for both of my monitors.

I'd be grateful for any hint as to how to specify the monitor to use in the newest version of OpenVR.

**Comment by TheDeveloperGuy** [comment]
Those properties are only used when using direct mode. If you're writing your own driver, it's unlikely that you're using a device that's been whitelisted for direct mode by nVidia or AMD. In that case you're going to be using extended mode and telling SteamVR which monitor you want to use via GetWindowBounds(). This has always (well more than the last 2 years anyway) been this way.

**Comment by Seneral** [comment]
Thank you very much for clarifying that! 
The docs said "Size and position that the window needs to be on the VR display." and I got a vrcompositor error saying "Invalid adapter output specified!" so I was fairly certain it needed to specify a display.
Still, for some reason it didn't work initially for me, and I don't know for sure why. 
I do know that I can trigger the displays the adapter error again by setting the adapter LUID and EDID VID&PID properties, but that can't be the only cause as the very same error showed up initially as well...
Anyway, thank you for clarifying this to me:)

Now, I am facing a different error I got already when setting it to virtual display (with 0,0 position)...
